deployment/legacy-migration/maintenance-worker/migration/neo4j/comments/comments.cql

```diff
@@ -55,6 +55,8 @@ ON CREATE SET
 c.content        = comment.content,
 c.contentExcerpt = comment.contentExcerpt,
 c.deleted        = comment.deleted,
+c.createdAt      = comment.createdAt.`$date`,
+c.updatedAt      = comment.updatedAt.`$date`,
```

Comment:
I suppose you have tested it.
Therefore I approve. 💫

But where is this `$date` coming from?

Reply:
the date would be coming from what is in the database in the `alpha`, I would guess... this is the same way we are setting the date for `Post`s